test/runtests.jl

```diff
@@ -23,7 +23,7 @@ wave = 1.e4 ./ x_inv_microns
 ref_values = [1.569, 1.337, 1.000, 0.751, 0.479, 0.282, 0.190, 0.114]
 
 for i=1:length(wave)
-    @test_approx_eq_eps ccm89(wave[i], 3.1) ref_values[i] 0.016*ref_values[i]
+    @test ccm89(wave[i], 3.1) ≈ ref_values[i] atol=0.016*ref_values[i]
```

Comment:
Now that you're using the new `isapprox` syntax, you can simplify the condition by using the relative tolerance in place of the absolute tolerance: `rtol=0.016`.  You should do the same for the other tests.

Reply:
Great! Thank you for the suggestions.